package/com.unity.formats.usd/Samples/ImportMaterials/ImportMaterialsExample.cs

```diff
@@ -44,31 +47,14 @@ public struct ShaderPair
             }
         }
 
-        private Transform m_cube;
-
-        void Update()
-        {
-            if (!m_cube)
-            {
-                return;
-            }
-
-            var rot = m_cube.localEulerAngles;
-            rot.y += Time.deltaTime * 10 + Mathf.Sin(Time.time) * .1f + .05f;
-            m_cube.localEulerAngles = rot;
-        }
```

Comment:
I quite liked the cube that showed off the imported material, what was the reasoning for removing it?

Reply:
Yeah i guess i could add it back in to show it off